programs/account-compression/tests/merkle_tree_tests.rs

```diff
@@ -696,9 +696,40 @@ async fn functional_1_initialize_state_merkle_tree_and_nullifier_queue(
         merkle_tree_pubkey,
     )
     .await;
+
+    assert_eq!(merkle_tree.deserialized().index, 1);
```

Comment:
seeing that the asserts in the address Merkle tree tests and this test are essentially the same maybe it makes sense to do an assert function that take a concurrent Merkle tree struct and a struct with expected values as inputs that can be used in both cases.
wdyt?

Reply:
I agree, will do that.

Reply:
thanks!